WORKSPACE

```diff
@@ -102,4 +102,13 @@ container_pull(
     repository = "distroless/cc",
 )
 
+container_pull(
+    name = "bionic",
```

Comment:
Could we avoid referring to a specific version in the container name? This should be "ubuntu", IMHO.

Reply:
Ok.

Reply:
I thought this was renamed already? Or is the name enforced in the upstream repo, somehow?

Reply:
It's not, but this is just an internal name. It's better than "ubuntu" since we can pull other ubuntus if necessary.